Add optional preview to generators (#587)
github/view_component: docs/index.md

bin/rails generate component Example title content --template-engine slim
```

> Rails preview can also be generated with the `--preview` option:

Reviewer: ```suggestion
To generate a [preview](#previewing-components), pass the `--preview` option:
```
